packages/flutter/test/material/snack_bar_test.dart

```diff
@@ -482,4 +482,49 @@ void main() {
     expect(closedReason, equals(SnackBarClosedReason.timeout));
   });
 
+  testWidgets('SnackBar default display duration test', (WidgetTester tester) async {
+    int snackBarCount = 0;
+    const Key tapTarget = const Key('tap-target');
+    await tester.pumpWidget(new MaterialApp(
+        home: new Scaffold(
+            body: new Builder(
+                builder: (BuildContext context) {
+                  return new GestureDetector(
+                      onTap: () {
+                        snackBarCount += 1;
+                        Scaffold.of(context).showSnackBar(new SnackBar(
+                            content: new Text('bar$snackBarCount')
+                        ));
+                      },
+                      behavior: HitTestBehavior.opaque,
+                      child: new Container(
+                          height: 100.0,
+                          width: 100.0,
+                          key: tapTarget
+                      )
+                  );
+                }
+            )
+        )
+    ));
+
+    expect(find.text('bar1'), findsNothing);
```

Comment:
This doesn't really test the default duration.  You'll want to use
```
tester.pump(duration)
```
to verify the timing - see https://docs.flutter.io/flutter/flutter_test/WidgetTester/pump.html

Reply:
also this is in the wrong PR 📦